extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -37,10 +37,42 @@
     @ConfigItem
     public List<String> topics;
 
+    /**
+     * The schema registry key.
+     * e.g. to diff between different registry impls / instances
```

Comment:
Perhaps clarify the value to be used for Apicurio?

Reply:
OK, will do.